meta-balena-common/recipes-core/initrdscripts/files/cryptsetup

```diff
@@ -18,12 +26,41 @@ cryptsetup_enabled() {
 }
 
 cryptsetup_run() {
+    # Die if anything fails here
+    set -e
```

Comment:
Why not move this to the whole script? is anything else failing acceptable?

Reply:
If I move it out of the function it would only apply to the function definition, not the actual execution. I was assuming this is disabled by default because otherwise any random failure in any initrd script would make the system unbootable - which is, however, exactly what I am trying to achieve for this particular function. I agree it is not nice, if you prefer I can add `if` around every command, it just feels it adds a lot of slack to the script, that is why I went for the `set -e` shortcut.

Reply:
leave it as is @mtoman it's just a personal preference to set it for the whole script - I don't know why it wasn't done that way - and explicitely || true those places that may fail - but it's definitely out of scope for this change, so your approach makes total sense.